whitehall/config/deploy.rb

```diff
@@ -61,13 +60,7 @@
 
     task :rsync_local_assets, roles => :web, :except => { :no_release => true } do
       find_servers.each do |server|
-        if File.exist?("#{strategy.local_cache_path}/public/government")
-          puts run_locally "cd #{strategy.local_cache_path} && rsync -aK ./public/government/ #{user}@#{server}:#{shared_path}/assets/;"
-        end
-
-        if File.exist?("#{strategy.local_cache_path}/public/assets")
-          puts run_locally "cd #{strategy.local_cache_path} && rsync -aK ./public/assets/ #{user}@#{server}:#{latest_release}/public/assets/;"
-        end
+        puts run_locally "cd #{strategy.local_cache_path} && rsync -aK ./public/assets/ #{user}@#{server}:#{shared_path}/assets/;"
```

Comment:
```suggestion
        puts run_locally "cd #{strategy.local_cache_path} && rsync -aK ./public/assets/ #{user}@#{server}:#{shared_path}/assets/"
```

I see it was there before, but do we actually need this `;`?

Reply:
Wouldn't have thought so, good catch. I'll give it a quick spin without just in case it surprises us with anything.